service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/CaseInitiationController.java

```diff
@@ -49,10 +48,13 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmitEvent(
 
     @PostMapping("/submitted")
     public void handleSubmittedEvent(
-        @RequestHeader(value = "authorization") String authorization,
-        @RequestHeader(value = "user-id") String userId,
         @RequestBody CallbackRequest callbackRequest) {
 
-        applicationEventPublisher.publishEvent(new InitiatedCaseEvent(callbackRequest, authorization, userId));
+        CaseDetails caseDetails = callbackRequest.getCaseDetails();
+        String caseId = Long.toString(caseDetails.getId());
+        String caseLocalAuthority = (String) callbackRequest.getCaseDetails().getData()
+            .get("caseLocalAuthority");
```

Comment:
this can just be caseDetails.getData... now

Reply:
:D